analyzers/src/SonarAnalyzer.CSharp/Rules/Hotspots/RequestsWithExcessiveLength.cs

```diff
@@ -41,14 +41,19 @@ public sealed class RequestsWithExcessiveLength : RequestsWithExcessiveLengthBas
         protected override bool IsInvalidRequestFormLimits(AttributeSyntax attribute, SemanticModel semanticModel) =>
             attribute.IsKnownType(KnownType.Microsoft_AspNetCore_Mvc_RequestFormLimitsAttribute, semanticModel)
```

Comment:
Actually, it may not be obvious on how to move the logic in `IsKnownType` , so we should leave that for a separate PR/issue.

Here you can just check the name with the expected attribute name. An better than using contains you can just compare the two attribute names (with or without the Attribute suffix). And given you'd do this across multiple places, extract a method for it.